src/windows/windows.es6

```diff
@@ -401,9 +401,6 @@ export const init = function($parentObj) {
    /* wrap jquery ui dialog('destory') to fire an event */
    var original = $.fn.dialog;
    $.fn.dialog = function(cmd) {
-      if(!this.data('ui-dialog')) {
-         return this;
-      }
```

Comment:
@4p00rv Just keep these 3 lines, i do think reverting this PR is the best solution. 👍